src/_data/LiveData.js

```diff
@@ -28,6 +28,13 @@ const handlers = {
 
 export const api = new LiveApi({ language: 'EN' });
 
+const subscribeToSelectedSymbol = () => {
+    loadedStorePromise.then(st => {
```

Comment:
LiveData should not be aware of the store that much, this needs to be injected from parent.